readme.md

```diff
@@ -8,8 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-
-## Languages (854 total)
+## Languages (855 total)
```

Comment:
By merging in the master branch, the language count still needed to be incremented, and now does.

Reply:
Done